File: python/pytorch/train/resnet18/train_cifar100.py
from python.pytorch.models.resnet18 import run_experiment

if __name__ == "__main__":
    run_experiment(
        dataset_path="data/cifar100_png",
        output_file="resnet18_cifar100.csv",
        checkpoint_path="checkpoints/resnet18_cifar100.pth",
        img_size=(32, 32),
        grayscale=False
    )
    
    
# PYTHONPATH=. python3 python/pytorch/train/resnet18/train_cifar100.py

File: R/train/resnet18/train_cifar100.r
source("R/models/resnet18.r")
cat("Script avviato!\n")
run_experiment(
  dataset_path = "/home/pampaj/DeepGreen/data/cifar100_png",
  checkpoint_path = "R/checkpoints/resnet18_cifar100_r.pt",
  img_size = c(32, 32),
  grayscale = FALSE
)